src/main/resources/application.yaml

```
@@ -20,11 +20,35 @@ spring:
+        tags-sorter: alpha
+        doc-expansion: none
+    api-docs:
+        path: /api-docs
+        enabled: true
+    show-actuator: false
+    model-and-view-allowed: false
 
 # Server Configuration
 server:
-    port: 8080
+    port: 8080
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)